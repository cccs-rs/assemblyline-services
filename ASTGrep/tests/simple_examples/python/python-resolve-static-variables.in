var1 = "evil code"

print(var1)