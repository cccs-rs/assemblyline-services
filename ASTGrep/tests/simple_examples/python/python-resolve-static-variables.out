var1 = "evil code"

print("evil code")